packages/forms/src/UIForm/fields/Select.js

```diff
@@ -0,0 +1,82 @@
+import React, { PropTypes } from 'react';
+import FieldTemplate from './FieldTemplate';
+
+function getSelectedOptions(select, multiple) {
+	if (multiple) {
+		return Array.prototype.slice.call(select.options)
+			.filter(option => option.selected)
+			.map(option => option.value);
+	}
+
+	return select.value;
+}
+
+export default function Select({ id, isValid, errorMessage, onChange, schema, value }) {
+	const { autoFocus, description, disabled, placeholder, readOnly, title } = schema;
+
+	const multiple = schema.schema.type === 'array' && schema.schema.uniqueItems;
+
+	return (
+		<FieldTemplate
+			description={description}
+			errorMessage={errorMessage}
+			id={id}
+			isValid={isValid}
+			label={title}
+			labelAfter
+		>
+			<select
+				id={id}
+				multiple={multiple}
+				autoFocus={autoFocus}
+				className="form-control"
+				disabled={disabled}
+				onChange={
+					event => onChange(event, schema, getSelectedOptions(event.target, multiple))
+				}
+				readOnly={readOnly}
+				value={value}
+			>
+				<option disabled>{placeholder}</option>
+				{
+					schema.titleMap && schema.titleMap.map((option, index) => {
+						const optionProps = {
+							key: index,
+							value: option.value,
+						};
+						return (
+							<option
+								{...optionProps}
+							>
+								{option.name}
+							</option>
+						);
+					})
+				}
+			</select>
+		</FieldTemplate>
+	);
+}
+
+if (process.env.NODE_ENV !== 'production') {
+	Select.propTypes = {
```

Comment:
PropTypes not marked as required, without default value could lead in actual implementation to runtime error